rabbitmq/metadata.csv

```diff
@@ -78,223 +78,226 @@ rabbitmq.overview.messages.publish_in.count,gauge,,message,,Count of messages pu
+rabbitmq.channel.messages.confirmed.count,count,,,,[OpenMetricsV2] Total number of messages published into an exchange and confirmed on the channel,0,rabbitmq,channel messages confirmed,
+rabbitmq.channel.messages.delivered.ack.count,count,,,,[OpenMetricsV2] Total number of messages delivered to consumers in manual acknowledgement mode,0,rabbitmq,channel messages delivered ack,
+rabbitmq.channel.messages.delivered.total.count,count,,,,[OpenMetricsV2] Total number of messages delivered to consumers in automatic acknowledgement mode,0,rabbitmq,channel messages delivered total,
+rabbitmq.channel.messages.published.count,count,,,,[OpenMetricsV2] Total number of messages published into an exchange on a channel,0,rabbitmq,channel messages published,
+rabbitmq.channel.messages.redelivered.total.count,count,,,,[OpenMetricsV2] Total number of messages redelivered to consumers,0,rabbitmq,channel messages redelivered total,
+rabbitmq.channel.messages.unacked,gauge,,,,[OpenMetricsV2] Delivered but not yet acknowledged messages,0,rabbitmq,channel messages unacked,
+rabbitmq.channel.messages.uncommitted,gauge,,,,[OpenMetricsV2] Messages received in a transaction but not yet committed,0,rabbitmq,channel messages uncommitted,
+rabbitmq.channel.messages.unconfirmed,gauge,,,,[OpenMetricsV2] Published but not yet confirmed messages,0,rabbitmq,channel messages unconfirmed,
+rabbitmq.channel.messages.unroutable.dropped.count,count,,,,[OpenMetricsV2] Total number of messages published as non-mandatory into an exchange and dropped as unroutable,0,rabbitmq,channel messages unroutable dropped,
+rabbitmq.channel.messages.unroutable.returned.count,count,,,,[OpenMetricsV2] Total number of messages published as mandatory into an exchange and returned to the publisher as unroutable,0,rabbitmq,channel messages unroutable returned,
+rabbitmq.channel.prefetch,gauge,,,,[OpenMetricsV2] Total limit of unacknowledged messages for all consumers on a channel,0,rabbitmq,channel prefetch,
+rabbitmq.channel.process_reductions.count,count,,,,[OpenMetricsV2] Total number of channel process reductions,0,rabbitmq,process reductions,
+rabbitmq.channels.closed.count,count,,,,[OpenMetricsV2] Total number of channels closed,0,rabbitmq,channels closed,
+rabbitmq.channels.opened.count,count,,,,[OpenMetricsV2] Total number of channels opened,0,rabbitmq,channels opened,
+rabbitmq.channels,gauge,,,,[OpenMetricsV2] Channels currently open,0,rabbitmq,channels,
+rabbitmq.cluster.vhost_status,gauge,,,,[OpenMetricsV2] Whether a given vhost is running,0,rabbitmq,cluster vhost status,
+rabbitmq.cluster.exchange_bindings,gauge,,,,[OpenMetricsV2] umber of bindings for an exchange. This value is cluster-wide.,0,rabbitmq,cluster exchange bindings,
+rabbitmq.cluster.exchange_name,gauge,,,,[OpenMetricsV2] Enumerates exchanges without any additional info. This value is cluster-wide. A cheaper alternative to `exchange_bindings`,0,rabbitmq,cluster exchange bindings,
```

Comment:
What is meant by `cheaper alternative`? Is this in context of rabbitmq native monitoring? We could probably leave part out for our check

Reply:
From rmq docs in general, by "cheaper" I think they mean the rmq node computes this metric faster.